exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -183,6 +183,15 @@ def _translate_to_zipkin(self, spans: Sequence[Span]):
                     "otel.instrumentation_library.version"
                 ] = span.instrumentation_info.version
 
+            if span.status is not None:
+                zipkin_span["tags"][
+                    "ot.status_code"
```

Comment:
Hey there - these tags don't quite look correct because they're `ot`. The referenced spec was added several months ago (https://github.com/open-telemetry/opentelemetry-specification/pull/380) before the acronym was officially finalized as `otel`. I have sent https://github.com/open-telemetry/opentelemetry-specification/pull/967 to fix this.

I'd recommend either proactively using `otel` (it's pretty obvious that's what it should be, see lines 180 / 183 in this file) or waiting a bit if you'd like to see the update in the spec merged. As it stands, if this merges it'll need to be updated right away anyways.